client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
```

Comment:
Yeah, that's an old habit of mine.
I like to group dependencies on the basis of _their purpose_; in this case I put that extra line to separate react, from the mocking/assertion libraries.

Since you don't have this convention, to let that extra line does not make sense in this case.